src/frontend/js/widgets/Dashboard/components/TeacherOrganizationDashboardSidebar/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { defineMessages, useIntl } from 'react-intl';
+import { generatePath, useParams } from 'react-router-dom';
+import { useMemo } from 'react';
+import { TeacherDashboardPaths } from 'widgets/Dashboard/utils/teacherRouter';
+import { DashboardSidebar } from 'widgets/Dashboard/components/DashboardSidebar';
+import { getDashboardRouteLabel, getDashboardRoutePath } from 'widgets/Dashboard/utils/routers';
+
+const messages = defineMessages({
+  header: {
+    id: 'components.TeacherProfileDashboardSidebar.header',
+    description: 'Title of the dashboard sidebar',
+    defaultMessage: 'TODO ORGANIZATION NAME',
+  },
+  subHeader: {
+    id: 'components.TeacherProfileDashboardSidebar.subHeader',
+    description: 'Sub title of the dashboard sidebar',
+    defaultMessage: 'You are on the organization dashboard',
+  },
+});
+
+export const TeacherOrganizationDashboardSidebar = () => {
+  const intl = useIntl();
+  const getRoutePath = getDashboardRoutePath(intl);
+  const getRouteLabel = getDashboardRouteLabel(intl);
+  const params = useParams();
+
+  const links = useMemo(
+    () =>
+      [TeacherDashboardPaths.ORGANIZATION_COURSES, TeacherDashboardPaths.ORGANIZATION_MEMBERS].map(
+        (path) => ({
+          to: generatePath(
+            getRoutePath(path, {
+              organizationId: ':organizationId',
+            }),
+            params,
+          ),
+          label: getRouteLabel(path),
+        }),
+      ),
+    [],
+  );
+  const settingsUrl = generatePath(
+    getRoutePath(TeacherDashboardPaths.ORGANIZATION_SETTINGS, {
+      organizationId: ':organizationId',
+    }),
+    params,
+  );
+
+  return (
+    <DashboardSidebar
+      menuLinks={links}
+      settingsUrl={settingsUrl}
+      header={messages.header}
+      subHeader={messages.subHeader}
```

Comment:
I think you should use `intl.formatMessage` to pass down texts because as you are currently implemeting things, the sub component will not be able to pass variables if needed.

Reply:
good catch, i'll change that.